src/js/server/devpmt-datasources.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.devpmt");
+var fs = require("fs");
+var JSON5 = require("json5");
+
+gpii.devpmt.loadNPSet = function (prefsetDir, prefsetName, ontologyHandler) {
+    var elod = JSON5.parse(fs.readFileSync(
+        __dirname + prefsetDir + prefsetName + ".json"));
```

Comment:
Should use .json5 extension. The naming of this file is odd in that it includes no DataSources

Reply:
the remaining things in this file now are datasources and json5